docs/en/cookbook/working-with-datetime.rst

```diff
@@ -67,84 +67,20 @@ The problem is simple. Not a single database vendor saves the timezone, only the
 However with frequent daylight saving and political timezone changes you can have a UTC offset that moves
 in different offset directions depending on the real location.
 
-The solution for this dilemma is simple. Don't use timezones with DateTime and Doctrine 2. However there is a workaround
+The solution for this dilemma seems simple. Don't use timezones with DateTime and Doctrine 2. However there is a workaround
 that even allows correct date-time handling with timezones:
 
-1. Always convert any DateTime instance to UTC.
```

Comment:
I suggest adding a section that links to your article about the original problem. Most people will try looking for a simple "convert to UTC" solution, and they will mess it up.